### ISSUE TITLE
create a new branch for Symfony2.0 compat

### ISSUE THREAD

**Issue by lsmith77** [FriendsOfSymfony/FOSFacebookBundle#130]
we should create a new branch from master for 2.0 compat using composer. not sure what to call it though.

**Comment by schmittjoh** [comment]
Don't we already have broken compat with 2.0, and a separate branch for it? Or what is the 2.0 branch good for?

**Comment by lsmith77** [comment]
yes .. this is currently handled by the 2.0 branch which however is out of sync. more importantly it doesn't follow the conventions we defined here https://github.com/FriendsOfSymfony/friendsofsymfony.github.com/issues/27

**Comment by schmittjoh** [comment]
I remember that discussion, but was a decision taken on which way to go? I'm at least not aware of what conventions we do have.

More importantly, I think we cannot branch from master anymore since BC has already been broken, or someone would have to revert some of the commits after branching to make it compatible with 2.0 again.

**Comment by lsmith77** [comment]
yeah .. this is what i did for FOSRestBundle .. aka reverting the relevant changes when i created the new branch

**Comment by stof** [comment]
reverting does not seem clean to me. If we decide to use a merge workflow, the outdated branch could be an issue. If we cherry-pick, it will work (we simply need to cherry-pick the relevant commits)

However, one thing is sure: we need to rename the branch to use a version number of the bundle.

**Comment by lsmith77** [comment]
i think reverting is cleaner .. but if you guys prefer we can also merge master into 2.0

**Comment by schmittjoh** [comment]
Reverting is the same like cherry-picking to me. I'd prefer that.

On Fri, Mar 16, 2012 at 7:56 PM, Lukas Kahwe Smith <
reply@reply.github.com
